src/main/java/frc/robot/subsystems/climber/ClimberReal.java

```diff
@@ -0,0 +1,50 @@
+package frc.robot.subsystems.climber;
+
+import com.ctre.phoenix6.BaseStatusSignal;
+import com.ctre.phoenix6.StatusSignal;
+import com.ctre.phoenix6.configs.TalonFXConfiguration;
+import com.ctre.phoenix6.controls.Follower;
+import com.ctre.phoenix6.hardware.TalonFX;
+import com.ctre.phoenix6.signals.NeutralModeValue;
+import edu.wpi.first.units.measure.Angle;
+import edu.wpi.first.wpilibj.DigitalInput;
+import frc.robot.Constants;
+
+
+
+public class ClimberReal implements ClimberIO {
+    private final TalonFX climberMotorLeft =
+        new TalonFX(Constants.Climb.LEFT_TALON_FX_ID, "canivore"); // Kraken
+    private final TalonFX climberMotorRight =
+        new TalonFX(Constants.Climb.RIGHT_TALON_FX_ID, "canivore"); // Kraken Motor
+    private final TalonFXConfiguration rightConfig = new TalonFXConfiguration(); // Kraken Config
+    private final TalonFXConfiguration leftConfig = new TalonFXConfiguration(); // Kraken Config
+    private StatusSignal<Angle> climbMotorPosition = climberMotorLeft.getPosition();
```

Comment:
If left is following right, probably should use right's encoder since it's the master.